locales/core/de.json

```diff
@@ -0,0 +1,207 @@
+  "gotIt": "Alles klar",
+  "helpLights": "Hilf uns, die Lichter anzubehalten",
+  "holder": {
+    "confirmPassword": "Bestätige dein neues Passwort",
+    "email": "E-Mail",
+    "enterPassword": "Gib ein neues Passwort ein",
+    "enterToken": "Reset Token eingeben",
+    "firstName": "Vorname",
+    "lastName": "Nachname",
+    "password": "Passwort",
+    "passwordRepeat": "Passwort wiederholen",
+    "radio": "Suche auf reddit.com",
+    "search": "Suche nach Titel / Album / Playlist / Kanal"
+  },
+  "hot": "Beliebt",
+  "lastFm": {
+    "remember": "Für die aktuelle Warteschleife merken",
+    "scrobble": "Scrobble zu last.fm"
+  },
+  "learnMoreAction": "Klicke hier, um mehr über unsere Preise zu erfahren",
+  "like": "Like",
+  "like_plural": "Likes",
+  "live": "Live",
+  "loadMore": "Mehr laden",
+  "loading": "Wird geladen",
+  "logIn": "Einloggen",
+  "loggingOut": "Du wirst ausgeloggt... Einen Moment",
+  "loginContinue": "Einloggen, um fortzufahren",
+  "loginNotification": "Hey {{name}}! Willkommen zurück!",
+  "mixAll": "Alle Mixen",
+  "month": "Monat",
+  "new": "Neu",
+  "newUser": "Neu bei Headset?",
+  "noType": {
+    "explainChannels": "Ein Kanal auf YouTube ist wie die Homepage für einen Account. Du kannst so vielen folgen, wie du willst. Achte in den Suchergebnissen auf den Tag <1>Kanal</1>!",
+    "explainCollections": "Sammlungen sind benutzerdefinierte Playlists, die mit einzelnen Titeln gefüllt sind. Erstelle eine Sammlung deiner Favoriten, damit sie immer griffbereit sind!",
+    "explainPlaylists": "Eine Playlist ist eine Gruppierung von Videos auf den YouTube-Kanälen der Benutzer, die auf eine bestimmte Weise aufgelistet sind, damit du schnell die Inhalte findest, die dich interessieren. Achte in den Suchergebnissen auf den Tag <1>Playlist</1>!"
+  },
+  "notification": {
+    "description": "Benachrichtigung anzeigen, wenn ein neuer Song abgespielt wird",
+    "notification_plural": "Benachrichtigungen",
+    "wentWrong": "Etwas ist falsch gelaufen! Bitte versuche es erneut"
+  },
+  "ok": "Okay",
+  "oneMoment": "Einen Moment bitte...",
+  "optional": "Optional",
+  "partyShuffle": "Party Shuffle",
+  "partyShuffleTooltip": "Spiele einen Mix aus all deinen Favoriten in einer übersichtlichen Playlist ab",
+  "passwordReset": "Passwort zurücksetzen",
+  "passwordResetSuccess": "Erfolgreich! Ändere jetzt dein Passwort und du bist fertig!",
+  "pastHour": "Letzte Stunde",
+  "pinned": "Angepinnt",
+  "plan": "Plan",
+  "playNext": "Als Nächstes abspielen",
+  "playSimilar": "Ähnliche Songs abspielen",
+  "playlist": "Playlist",
+  "playlistDelete": "Aus deiner Playlist löschen",
+  "playlistDeleteCreator": "Diese Playlist wurde vom Ersteller gelöscht.",
+  "playlist_plural": "Playlists",
+  "preferences": "Einstellungen",
+  "radio": "Radio",
+  "redirecting": "Leite um...",
+  "related": "Ähnlich wie",
+  "repeat": {
+    "all": "Alle wiederholen",
+    "one": "Einen Song wiederholen",
```

Comment:
This is a little too long, can it be shortened?

Reply:
Changed it to "Dies wiederholen", short enough?

Reply:
This is long enough. Thanks!